conn2res/tasks.py

```diff
@@ -276,16 +269,18 @@ def name(self, name):
 
         self._name = name
 
-    def fetch_data(self, n_trials=None, horizon=None, **kwargs):
+    def fetch_data(self, n_trials=None, horizon_max=-20, win=30, **kwargs):
```

Comment:
As this class refers to a memory capacity task, I only define `horizon_max` and all horizon levels are used up to that value (i.e., it is a multioutput task)

Reply:
Perfect!